src/Microsoft.AspNetCore.Mvc.ApiExplorer/ApiResponseFormat.cs

```diff
@@ -19,5 +21,27 @@ public class ApiResponseFormat
         /// The media type of the response.
         /// </summary>
         public string MediaType { get; set; }
+
+        /// <summary>
+        /// Gets or sets <see cref="ModelMetadata"/> for the <see cref="ResponseType"/> or null.
+        /// </summary>
+        /// <remarks>
+        /// Will be null if <see cref="ResponseType"/> is null or void.
+        /// </remarks>
+        public ModelMetadata ResponseModelMetadata { get; set; }
+
+        /// <summary>
+        /// Gets or sets the CLR data type of the response or null.
+        /// </summary>
+        /// <remarks>
+        /// Will be null if the action returns no response, or if the response type is unclear. Use
+        /// <c>ProducesAttribute</c> on an action method to specify a response type.
+        /// </remarks>
+        public Type ResponseType { get; set; }
```

Comment:
Let's change this to just `Type` and `ModelMetadata` for consistency with https://github.com/aspnet/Mvc/blob/dev/src/Microsoft.AspNetCore.Mvc.ApiExplorer/ApiParameterDescription.cs